include/sf33rd/Source/Game/BCD.h

```diff
@@ -0,0 +1,10 @@
+#ifndef BCD_H
+#define BCD_H
+
+#include "types.h"
+
+extern s16 bcdext; // size: 0x2, address: 0x578A9C
```

Comment:
These comments aren't needed anymore
```suggestion
extern s16 bcdext;
```

Reply:
removed!